packages/react/src/SubdomainNavBar/SubdomainNavBar.module.css

```diff
@@ -154,15 +154,22 @@
     /* overflow: hidden; */
   }
 }
+
+@media screen and (min-width: 750px) {
```

Comment:
Could we use our standard `768px` breakpoint or does it need to be exactly `750px`?